about.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- Using font-awesome website for icons-->
+    <link rel="stylesheet"
+    href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
+    integrity="sha256-46qynGAkLSFpVbEBog43gvNhfrOj+BmwXdxFgVK/Kvc=" crossorigin="anonymous" />
+    <!--Using google fonts-->
+    <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@1,300&family=Viaoda+Libre&display=swap" rel="stylesheet">
+    
+    <!-- link to index-->
+    <href class="nav-link"><a href="index.html">Back to Main Page!</a></li>
```

Comment:
`href` is not a tag.  Did you mean to make it an `li`.  Also this should be inside the `body`.

```suggestion
    <li ><a href="index.html">Back to Main Page!</a></li>
```